src/components/AccordionRow/AccordionRow.tsx

```diff
@@ -12,7 +12,9 @@ type Props = {
   /**
    * Child node(s) that can be nested inside component.
    */
-  children: React.ReactNode;
+  children:
```

Comment:
Isn't this what lines 7-9 and 26-31 already doing? ... or supposed to be doing? The types for children appear to be mixed up.

Reply:
Ya you're right, types inherited after removing the duplicate